src/Sparrow/Utils/WeakTypeCache.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Buffers;
+using System.Threading;
+using Sparrow.Binary;
+
+namespace Sparrow.Utils
+{
+    public class WeakTypeCache<T> : IDisposable
```

Comment:
I don't see any usage of this class. Was it supposed to be used somewhere?

Reply:
No, it was moved to Micro.Benchmark. No use for it, removed.